docs/usage_guide.rst

```diff
@@ -463,6 +463,25 @@ In case of an exchange providing a more granular permissions scheme (e.g. Coinba
 
 You may as well try creating an API key with the minimum read-related permissions, then adding it in rotki and finally checking that the connection was successful and data was loaded as expected. Otherwise, try again adding more read-related premissions.
 
+Binance / Binance US
+--------------
+
+Binance API is engineered in a way that makes really slow to query information for trades since every possible market pair has to be queried. This process can also fail since many requests have to be made to binance's servers and some rate limits could apply.
```

Comment:
```suggestion
Binance API is engineered in a way that makes it really slow to query information for trades since every possible market pair has to be queried. This process can also fail since many requests have to be made to binance servers and rate limits may apply.
```